test/FunctionsV2/StorageProviderSelectionTests.cs

```diff
@@ -4,77 +4,174 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using DurableTask.Core;
 using Microsoft.Azure.WebJobs.Extensions.DurableTask;
 using Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests;
-using Microsoft.Azure.WebJobs.Host.TestCommon;
 using Moq;
 using Xunit;
-using Xunit.Abstractions;
 
 namespace WebJobs.Extensions.DurableTask.Tests.V2
 {
     public class StorageProviderSelectionTests
     {
-        static StorageProviderSelectionTests()
-        {
-            AzureStorageDurabilityProviderFactoryMock = new Mock<IDurabilityProviderFactory>();
-            AzureStorageDurabilityProviderFactoryMock.Setup(f => f.Name).Returns("AzureStorage");
+        public StorageProviderSelectionTests() { }
 
-            MicrosoftSQLDurabilityProviderFactoryMock = new Mock<IDurabilityProviderFactory>();
-            MicrosoftSQLDurabilityProviderFactoryMock.Setup(f => f.Name).Returns("MicrosoftSQL");
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        [InlineData("AzureStorage")]
+        public void SelectingAzureStorageStorageProvider(string storageProvider)
```

Comment:
nit: If you parameterize the method, a more specific name is helpful. Maybe something like `StorageProviderTypeSpecified_CorrectStorageProviderFactoryUsed)`